### ISSUE TITLE
REVISE Impaired lymphocyte transformation with phytohemagglutinin  HP:0003347

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11504]
This term was actually a synonym of [Decreased PHA-induced T-cell proliferation](https://hpo.jax.org/browse/term/HP:0025834)

New label: Impaired phytohemagglutinin-induced T lymphocyte transformation
New parent:Decreased mitogen-induced T-cell proliferation  HP:0031381

New Def: A reduced rate of T lymphocyte transformation in response to in vitro stimulation to the mitogen phytohemagglutinin (PHA). Following PHA stimulation, T cells normally undergo morphological and biochemical alterations that reflect the transformation into lymphoblasts. There are several methods for quantifying this effect including measuring the uptake of the radioactive marker 3H-TdR, methyl thiazolyl tetrazolium colorimetric analysis (MTT assay), and morphological examination under the microscope or using a hematology analyzer. Various types of stimulation index compare the amount of proliferation between treated and control cells. An impaired test refers to a result in which the amount of stimulation is subnormal.
PMID: 36393975

**Comment by adamklocperk** [comment]
Hi @pnrobinson personally I like the old name and description of HP:0031381 better and would keep that one, as is, as the only term and merge the disease annotations. https://pmc.ncbi.nlm.nih.gov/articles/PMC9913443/ could be used as a resource, if need be, though you know my feeling on the PMID annotations :)

This HP:0003347 term is, therefore, in my view, superfluous. That being said, the new name and description are well-written and I don't see any issues with it.

**Comment by pnrobinson** [comment]
@adamklocperk 
Implemented as suggested. The remaining term id is HP:0025834
